src/classes/configuration_io.cpp

```diff
@@ -100,7 +100,10 @@ bool Configuration::deserialise(LineParser &parser, const CoreData &coreData, do
      */
     if (parser.getArgsDelim(LineParser::Defaults) != LineParser::Success)
         return false;
-    appliedSizeFactor_ = parser.argd(3);
+    if (parser.argd(3) == 1.0)
+        appliedSizeFactor_ = std::nullopt;
+    else
+        appliedSizeFactor_ = parser.argd(3);
```

Comment:
Comparing real values directly is always a bit dangerous.  Although this should be fine, I might suggest reversing the logic and doing:
```suggestion
    if (parser.argd(3) > 1.0)
        appliedSizeFactor_ = parser.argd(3);
    else
        appliedSizeFactor_ = std::nullopt;
```